core/src/foundations/tokens.preview.html

```diff
@@ -1725,6 +1727,10 @@ <h1>Space Tokens</h1>
             <div class="space-token" style="margin: 32px">-space-800 (32px)</div>
           </div>
 
+          <div class="spacing-wrapper">
+            <div class="space-token" style="margin: 34px">-space-850 (34px)</div>
```

Comment:
Why don't these use the classes like `.ionic-margin-space-850`?

Reply:
At the time I wanted to have this file abstract of the other ones, to be as `pure` as possible. 
But it makes sense, I think we can do that in a future improvement to this 👍

Reply:
That's fine with me!